.github/pull_request_template.md

```diff
@@ -0,0 +1,41 @@
+NOTICE: Delete the sections that do not apply to your PR, and fill out the section that does.
+(You can open a PR to add or improve a section, if you find them lacking!) 
+
+----------------------
+**Content (Artwork / Missions / Jobs)**
+
+## Summary
+{{summarize your content! Include links to related issues, in-game screenshots, etc.}}
+
+## Save File
+This save file can be used to play through the new mission content:
+{{attach a save file that allows people to easily test your added mission content or see your new in-game art}}
+
+## PR Checklist
+ - [ ] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified
+ - [ ] I uploaded the necessary image, blend, and texture assets here: {{insert link to assets}}
+ - [ ] I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: {{insert PR link}}
+  
+  
+-----------------------
+**Bugfix:** This PR addresses issue #{{insert number}}
```

Comment:
There are standard Github keywords to automatically close the issue at merge of the PR. Would be nice to use those here.

Reply:
I think I prefer a manual close for the linked issues, as they may not be fully addressed by any one PR. Those with the ability to merge can make edits to the post as needed, though.